### ISSUE TITLE
The command for opening the clojure-lsp log file doesn't (always?) work

### ISSUE THREAD

**Issue by bpringe** [BetterThanTomorrow/calva#2269]
It doesn't work for me on MacOS. I think it hasn't worked in a while for me. It seems it should work even if the log path is set dynamically by clojure-lsp since it uses the log path reported from the server's returned info. Either Calva is doing something incorrectly or clojure-lsp isn't returning the right path.

**Comment by PEZ** [comment]
It could also be that the log file has grown beyond 50Mb and we fail to catch the error from VS Code refusing to open it.

**Comment by bpringe** [comment]
Hmm... maybe to have a more reliable solution, we should just print the path somewhere for the user and let them open it however they want to.

**Comment by PEZ** [comment]
Indeed. Command-clicking the path will open it in VS Code, and probably display a proper error message.
